mutornadomon/config.py

```diff
@@ -28,4 +30,7 @@ def initialize_mutornadomon(tornado_app=None, publisher=None, publish_interval=N
         web_collector = WebCollector(monitor, tornado_app)
         web_collector.start()
 
+    utilization_collector = UtilizationCollector(monitor, publish_interval)
```

Comment:
Take a look at the `_reset_ephemeral` method of `MuTornadoMon`. We could hook something similar in there, I think we just need a new structure as an attribute and method to manipulate it.

Basically, the idea is to flush/reset the metric when `.metrics` is accessed.